src/app.rs

```diff
@@ -941,6 +954,24 @@ impl App {
     pub(crate) fn modifiers(&self) -> ModifiersState {
         self.pressed_modifiers
     }
+
+    /// sets the active keybind set to be the first non-hidden keybind set in the list.
+    fn reset_active_keybind_set(&mut self) {
+        // TODO(HactarCE): why do we store different keybind sets for each puzzle if the sets are actually the same for puzzles of the same dimension?
+        // i didn't look at how keybind sets are modified but this seems kinda weird.
```

Comment:
keybind sets are indexed by family. also this comment will never be addressed lmao